app/views/idv/in_person/ssn/show.html.erb

```diff
@@ -1,88 +0,0 @@
-<%#
-Renders a page asking the user to enter their SSN or update their SSN if they had previously entered it.
-
-locals:
-* updating_ssn: true if the user is updating their SSN instead of providing it for the first time. This
```

Comment:
This is in the comments on both templates. It looks like you have refactored this into a lie.